src/components/jacobian_component/jacobian.sh

```diff
@@ -29,6 +29,11 @@ setup_jacobian() {
     sed -i -e "s:{START}:0:g" \
            -e "s:{END}:${nElements}:g" \
            -e "s:{InversionPath}:${InversionPath}:g" jacobian_runs/submit_jacobian_simulations_array.sh
+    if [ $MaxSimultaneousRuns -gt 0 ]; then
+	sed -i -e "s:{JOBS}:%${MaxSimultaneousRuns}:g" jacobian_runs/submit_jacobian_simulations_array.sh
+    else
+	sed -i -e "s:{JOBS}::g" jacobian_runs/submit_jacobian_simulations_array.sh
+    fi
```

Comment:
How does this handle the case where the number of perturbation sims is under the number of MaxSimultaneousRuns? Eg. 71 perturbation sims, but MaxSimultaneousRuns is set to 100? Would this cause a bug/crash of submit_jacobian_simulations_array.sh?

Reply:
I'm not sure how slurm would handle this. In commit https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/9629c549521385b0709e1aa33c1c158082f6ddc7 I put in an error check to ensure MaxSimultaneousRuns is less than the total number of runs (aka nElements).